.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,19 @@
 pvlib python pull request guidelines
 ====================================
 
-Thank you for your contribution to pvlib python!
+Thank you for your contribution to pvlib python! You may delete all of these instructions except for the list below.
 
-You may submit a pull request with your code at any stage of completion, however, before the code can be merged the following items must be addressed:
+You may submit a pull request with your code at any stage of completion.
 
- - [ ] Closes issue #xxxx
- - [ ] Fully tested. Added and/or modified tests to ensure correct behavior for all reasonable inputs. Tests must pass on the TravisCI and Appveyor testing services.
- - [ ] Code quality and style is sufficient. Passes ``git diff upstream/master -u -- "*.py" | flake8 --diff`` and/or landscape.io linting service.
- - [ ] New code is fully documented. Includes sphinx/numpydoc compliant docstrings and comments in the code where necessary.
+The following items must be addressed before the code can be merged. Please don't hesitate to ask for help if you're unsure of how to accomplish any of the items below:
+
+ - [ ] Closes #xxxx
```

Comment:
Why remove "issue"? I think that might confuse a newbie: Closes what?

Reply:
Agree

Reply:
The reasoning behind this change is that github will automatically close an issue if a pull request is merged that has "closes #xxxx", but github will not do that for "closes issue #xxxx". But now that I think about it I agree that the change is a net negative.